moz_telemetry/modules/moz_telemetry/normalize.lua

```diff
@@ -192,6 +192,13 @@ function os(name)
     return normalize_os_grammar:match(name)
 end
 
+local normalize_os_version_grammar =
+l.C(l.R"09"^1 * (l.P"." * l.R"09"^1)^-1 * (l.P"." * l.R"09"^1)^-1)
+
+function os_version(v)
+    if type(v) ~= "string" then v = "" end
```

Comment:
why not return nil

Reply:
Also note per [the schema](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/templates/include/telemetry/environment.1.schema.json#L611-L613) that this could come in as an integer, in that case we should stringify it directly.

Reply:
@washort please update the code to handle the case where os version is a number.